### PR TITLE
Remove the duplicate source

### PR DESCRIPTION
**Pull request by nkolev92**
## Bug

Fixes: 
Regression: No  
* Last working version:   
* How are we preventing it in future:   

## Fix

Details: Remove the duplicate feed.   

## Testing/Validation

Tests Added: No  
Reason for not adding tests:  
Validation:

### CODE REVIEW

**Comment by rrelyea** [comment]
would it be better to leave "vside" named source than "cps"?
(will it help prevent people from adding the same source twice?)

or should we have them sorted by URL...so people notice?
etc...

**Review by rrelyea — APPROVED** [review]

**Comment by nkolev92** [comment]
Changed it to vside.
Not sure if we want to do anything more than that. 

Personally I think we just need to be more careful when PRing changes.

**Review by dominoFire — APPROVED** [review]

File: NuGet.Config [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
```
Comment (dominoFire) [inline NuGet.Config]: space?

Comment (nkolev92) [reply NuGet.Config]: fixed.

**Pull request merged**